editor/editor_node.cpp

```diff
@@ -1001,6 +1001,11 @@ void EditorNode::_resources_reimported(const Vector<String> &p_resources) {
 		}
 	}
 
+	if (scenes.size() > 0) {
+		// Editor may crash when related animation is playing while re-importing GLTF scene, stop it in advance.
+		AnimationPlayerEditor::get_singleton()->get_player()->stop(true);
+	}
+
```

Comment:
I added the scene size check because I only want to stop the animation when a gltf is reloaded. But I might be wrong. Thanks for helping with the null check!